packages/safe-web3-lib/src/Web3Adapter.ts

```diff
@@ -99,15 +98,15 @@ class Web3Adapter implements EthAdapter {
     return getGnosisSafeProxyFactoryContractInstance(safeVersion, proxyFactoryContract)
   }
 
-  getContract(address: string, abi: AbiItem[]): any {
-    return new this.#web3.eth.Contract(abi, address)
+  getContract(address: string, abi: AbiItem | AbiItem[], options?: ContractOptions): any {
```

Comment:
Does this really return `any`? 👀

Reply:
It seems like the type is `'TODO:'` on web3's end. It's possible to use `ReturnType<typeof this.#web3.eth.Contract>` to infer it I think. I'll resolve this, however, as it is not properly typed by a third party.